CHANGES.md

```diff
@@ -1,5 +1,53 @@
+- [dagster-aws] `PipesECSClient` now attaches AWS ECS metadata to Dagster results produced during Pipes invocation and adds Dagster tags to the ECS task.
+- [dagster-aws] `PipesEMRClient` now attaches AWS EMR metadata to Dagster results produced during Pipes invocation.
+- [dagster-databricks] `PipesDatabricksClient` now attaches Databricks metadata to Dagster results produced during Pipes invocation and adds Dagster tags to the Databricks job.
+- [dagster-fivetran] Added `load_fivetran_asset_specs` function. It can be used with the `FivetranWorkspace` resource and `DagsterFivetranTranslator` translator to load your Fivetran connector tables as external assets in Dagster.
+- [dagster-looker] Errors are now handled more gracefully when parsing derived tables.
+- [dagster-sigma] Sigma assets now contain extra metadata and kind tags.
+- [dagster-sigma] Added support for direct workbook to warehouse table dependencies.
+- [dagster-sigma] Added `include_unused_datasets` field to `SigmaFilter` to disable pulling datasets that aren't used by a downstream workbook.
+- [dagster-sigma] Added `skip_fetch_column_data` option to skip loading Sigma column lineage. This can speed up loading large instances.
+- [dagster-sigma] Introduced an experimental `dagster-sigma snapshot` command, allowing Sigma workspaces to be captured to a file for faster subsequent loading.
+
+### Bugfixes
+
+- Fixed a bug in run status sensors where setting incompatible arguments `monitor_all_code_locations` and `monitored_jobs` did not raise the expected error. (Thanks, [@apetryla](https://github.com/apetryla)!)
+- Fixed an issue that would cause the label for `AutomationCondition.any_deps_match()` and `AutomationCondition.all_deps_match()` to render incorrectly when `allow_selection` or `ignore_selection` were set.
+- Fixed a bug which could cause code location load errors when using `CacheableAssetsDefinitions` in code locations that contained `AutomationConditions`
+- Fixed an issue where the default multiprocess executor kept holding onto subprocesses after their step completed, potentially causing `Too many open files` errors for jobs with many steps.
+- [ui] Fixed an issue introduced in 1.9.2 where the backfill overview page would sometimes display extra assets that were targeted by the backfill.
+- [ui] Fixed "Open in Launchpad" button when testing a schedule or sensor by ensuring that it opens to the correct deployment.
+- [ui] Fixed an issue where switching a user setting was immediately saved, rather than waiting for the change to be confirmed.
+- [dagster-looker] Unions without unique/distinct criteria are now properly handled.
+- [dagster-powerbi] Fixed an issue where reports without an upstream dataset dependency would fail to translate to an asset spec.
+- [dagster-sigma] Fixed an issue where API fetches did not paginate properly.
+
+### Documentation
+
+- Added an [Airflow Federation Tutorial](https://docs.dagster.io/integrations/airlift/federation-tutorial/overview).
```

Comment:
Can we just have an airlift section added to this release? This is what I'm thinking:
```
### Introducing: `dagster-airlift`

`dagster-airlift` is coming out of stealth, and we've added a bunch of airlift content to the docs site. See the initial Airlift RFC [here](https://github.com/dagster-io/dagster/discussions/25279).
- A full [Airflow migration tutorial](https://master.dagster.dagster-docs.io/integrations/airlift/tutorial/overview
- A tutorial on [federating between Airflow instances](https://master.dagster.dagster-docs.io/integrations/airlift/federation-tutorial/overview)
- A bunch of other Airflow-related content coming soon! We'd love for you to check it out, and post any comments / questions in the #airflow-migration channel in the Dagster slack.

Reply:
did some wordsmithing to make it match the tone of the changelog more closely and nested it under the New section. I'm going to see what the preview looks like to see if it looks ok. I don't know if we've done this special kind of callout for a library before, so we might need to mess with the formatting so it looks good